OsmAnd/src/net/osmand/plus/myplaces/tracks/filters/FiltersAdapter.kt

```diff
@@ -60,12 +63,18 @@ class FiltersAdapter(
 
 			FilterType.CITY -> {
 				val view = inflater.inflate(R.layout.filter_city_item, parent, false)
-				FilterCityViewHolder(view, nightMode)
+				FilterCityViewHolder(
+					activity.applicationContext as OsmandApplication,
```

Comment:
Better to add app as class variable in FiltersAdapter than use this cast in many places